### DISCUSSION TITLE
wallet and solo problem

### DISCUSSION THREAD

**Discussion by knightblackm** [Chia-Network/chia-blockchain#5506] [General]
i saw my wallet at power shell today;

Wallet height: 
Sync status: Synced
Balances, fingerprint: 
Wallet ID 1 type STANDARD_WALLET
   -Total Balance: 1e-10 xch (100 mojo)
   -Pending Total Balance: 1e-10 xch (100 mojo)
   -Spendable: 1e-10 xch (100 mojo)

**Comment by lvcivs** [comment]
What is the problem?

**Reply by knightblackm** [reply]
i didn't farm any block, but i have above chia coin on wallet(i used solo only, not uupool and hpool)

**Reply by lvcivs** [reply]
You probably got 100 mojo from the chia faucet.
